compose/material3/material3/build.gradle

```diff
@@ -110,7 +110,6 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
                 api(project(":compose:runtime:runtime"))
                 api(project(":compose:ui:ui-graphics"))
                 api(project(":compose:ui:ui-text"))
-                api(project(":graphics:graphics-shapes"))
```

Comment:
Should we remove it from publication?

Reply:
I would keep it, to make sure that we won't break it.

We can remove it after creating a release branch - we usually collect all libraries we plan to stabilize